lib/pure/net.nim

```diff
@@ -531,7 +531,10 @@ when defineSsl:
     ## The last two parameters specify the certificate file path and the key file
     ## path, a server socket will most likely not work without these.
     ## Certificates can be generated using the following command:
-    ## ``openssl req -x509 -nodes -days 365 -newkey rsa:1024 -keyout mycert.pem -out mycert.pem``.
+    ## ``openssl req -x509 -nodes -days 365 -newkey rsa:4096 -keyout mykey.pem -out mycert.pem``
+    ## or using ECDSA:
```

Comment:
This needs empty ``##``, otherwise it will be badly formatted.

Reply:
I've pushed a commit that fixes the formatting. (I checked the result of `nim doc` locally)